zerver/views/users.py

```diff
@@ -399,9 +415,16 @@ def avatar(
 
 
 def avatar_medium(
```

Comment:
Is this worth separating into something like `avatar_medium_by_email` & `avatar_medium_by_user_id` or is this reasonable enough?

Reply:
I think it's worth splitting for the same reason the other is: So that we can have documentation of "Get high-resolution avatar by ID" or "by email" as separate endpoints.

Reply:
Oh, I guess you do have the URLs split. I suspect the code might feel slightly nicer the other way, but this change is sufficient to be able to do the documentation work we'd like to do.